flatpak/requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
-lxml==4.5.2
-caldav==0.8.0
-dbus-python==1.2.8
+lxml==4.9.4
+caldav==1.3.9
+dbus-python==1.2.18
```

Comment:
I don't think this one is needed anymore. I removed `dbus-python` in https://github.com/getting-things-gnome/gtg/pull/1025, and it seems to run "just fine" (I haven't tested it extensively, but `import dbus` doesn't fail)

Reply:
Note that the Hamster plugin still uses this. https://github.com/getting-things-gnome/gtg/pull/676 is supposed to port from dbus-python to gio dbus. On a quick test in a flatpak SDK shell, `org.gnome.Sdk/x86_64/45` has `dbus-python` but current `org.gnome.Sdk/x86_64/master` doesn't.

Reply:
Ah, thanks

Reply:
Interesting! It looks like it got added with version 45 to the runtime.